backend/prompt_studio/prompt_studio_index_manager/views.py

```diff
@@ -25,3 +34,54 @@ def get_queryset(self) -> Optional[QuerySet]:
         if filter_args:
             queryset = IndexManager.objects.filter(**filter_args)
         return queryset
+
+    def get_indexed_data_for_profile(self, request) -> Response:
+        queryset = self.get_queryset()
+        org_id = UserSessionUtils.get_organization_id(request)
+        util = PromptIdeBaseTool(log_level=LogLevel.INFO, org_id=org_id)
+        result = {}
+
+        if queryset is not None:
+            # Perform operations on the queryset
+            # Example: iterate through queryset and print each object's id
+            for obj in queryset:
```

Comment:
What would br in queryset ? profiles, right? maximum 4 right?

Reply:
queryset will have index manager data for requested profile.